script.user.js

```diff
@@ -66,6 +66,10 @@ let ai = {rule: {}};
 let elSetup;
 let nonce;
 
+// parameter
```

Comment:
An ALL_CAPS is for constants that don't change. Use a normal camelCase.
The default should be stored in `cfg` and exposed in setup for the user to configure.